nixos/modules/programs/environment.nix

```diff
@@ -21,6 +21,8 @@ in
         NIXPKGS_CONFIG = "/etc/nix/nixpkgs-config.nix";
         PAGER = mkDefault "less -R";
         EDITOR = mkDefault "nano";
+        XDG_CONFIG_HOME = "$HOME/.config";
+        XDG_DATA_HOME = "$HOME/.local/share";
```

Comment:
Since these are the default values according to the XDG spec, should we really set them explicitly? Might be nicer not to clutter the environment.